tests/utils/inputparams/testparams.go

```diff
@@ -78,6 +78,14 @@ func GetUniqueVolumeName(volName string) string {
 	return normalizeVolumeName(volName + "_volume_" + GetRandomNumber())
 }
 
+// GetSharedVolumeName prepares unique volume name that is DNS compliant
+// Shared volumes launch docker service. Name of such a service must be
+// DNS compliant, hence volname should also be DNS compliant.
```

Comment:
Yes. There are certain conventions for DNS names. Eg:
Each name can be 3 - 63 char long
Each name can be made out of labels that can be separated by single period "."
Each label can contain letters, number and hyphen only

Another alternative could be to make change to plugin code and not use volume name in file server name entirely.